include/experimental/__p1673_bits/blas2_matrix_vector_product.hpp

```diff
@@ -123,6 +124,7 @@ template<class ElementType_A,
          extents<>::size_type ext_z,
          class Layout_z,
          class Accessor_z>
+  requires (Layout_A::template mapping<std::experimental::extents<numRows_A, numCols_A>>::is_always_unique())
```

Comment:
This is not actually part of the formal wording, though it does appear once in the proposal.  The Requirements in **linalg.algs.reqs** only asks for "unique layout," which implies `is_unique()`, not necessarily `is_always_unique()`.  I'll file an issue with the proposal to make this more clear -- I think what we want is `is_unique()`, with burden on the caller to check this.